From 3b8cf49e60fb4cfd5bf0894830f8c3c78ef1d299 Mon Sep 17 00:00:00 2001
From: Egor Zhyh <egor990095@gmail.com>
Date: Wed, 30 May 2018 19:02:19 +0200
Subject: [PATCH] font

---
 config.def.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/config.def.h b/config.def.h
index 82b1b09..777150f 100644
--- a/config.def.h
+++ b/config.def.h
@@ -5,7 +5,7 @@
  *
  * font: see http://freedesktop.org/software/fontconfig/fontconfig-user.html
  */
-static char *font = "Liberation Mono:pixelsize=12:antialias=true:autohint=true";
+static char *font = "SauceCodePro Nerd Font:pixelsize=14:antialias=true:autohint=true";
 static int borderpx = 2;
 
 /*
-- 
2.17.0

